<launch>

  <node pkg = "necst_ros3"
      name = "weather_B_container"
      type = "weather_B_container.py" />

  <node pkg = "necst_ros3"
      name = "weather_D_container"
      type = "weather_D_container.py" />

  <node pkg = "necst_ros3"
      name = "weather_cabin"
      type = "weather_cabin.py" />

  <node pkg = "necst_ros3"
      name = "weather_dome"
      type = "weather_dome.py" />

  <node pkg = "necst_ros3"
      name = "weather_outside_davis"
      type = "weather_outside_davis.py" />

  <node pkg = "necst_ros3"
      name = "weather_outside_ondotori"
      type = "weather_outside_ondotori.py" />

  <node pkg = "necst_ros3"
      name = "weather_press"
      type = "weather_press.py" />

  <node pkg = "necst_ros3"
      name = "weather_rain"
      type = "weather_rain.py" />

  <node pkg = "necst_ros3"
      name = "weather_wind"
      type = "weather_wind.py" />

</launch>
